src/command.rs

```diff
@@ -0,0 +1,304 @@
+// Copyright 2023 Helsing GmbH
```

Comment:
Now I've done all the hard work of reconciling with main... 😅 

More seriously though, the reason I refactored this is because `lib.rs` and `main.rs` have exact duplicates of the `cmd::install()` code, and now that this has grown more complex it's super iffy to keep the implementations separate. I _could_ fix this in a later PR (and if I'd known how much this would've conflicted with Kim's changes I'd have done so), but is it really worth reverting at this point? (maybe it is, but don't want to do this unless you feel strongly)

Reply:
Agreed, the diff is not really understandable. One reason for doing this is that `buffrs::build` reuses the install command now i think

Reply:
Fair enough. I'll avoid this in the future. For what it is worth the only function I touched here is `install()`. Well at least aside from your nits I actioned in the others.